pkg/volume/aws_ebs/aws_util.go

```diff
@@ -91,12 +92,17 @@ func (util *AWSDiskUtil) CreateVolume(c *awsElasticBlockStoreProvisioner) (aws.K
 	}
 	// Apply Parameters (case-insensitive). We leave validation of
 	// the values to the cloud provider.
+	var PutAdminAndUserRequestsTogetherParams volume.PutAdminAndUserRequestsTogetherParams
```

Comment:
Style nit - I think the variable name should start with a lower-case letter i.e. `var putAdminAnd...`